image/imagenone.go

```diff
@@ -0,0 +1,39 @@
+// +build !imagick
+// Stubs to make code compile and behave same if it has image supportr or not.
+
+package image
+
+// Image type for holding configuration
+type Image struct {
```

Comment:
Is it possible to not have the exact same `type Image struct` in the code twice?

Reply:
Separated type into its own file